src/ynEIGEN/EigenStrategyManager.sol

```diff
@@ -306,7 +347,16 @@ contract EigenStrategyManager is
                     asset,
                     strategies[asset].userUnderlyingView((address(node)))
                 );
-                stakedBalances[j] += strategyBalance;
+
+                uint256 strategyWithdrawalQueueBalance;
+                uint256 queuedShares = node.queuedShares(strategies[asset]);
+                if (queuedShares > 0) {
+                    strategyWithdrawalQueueBalance = toUserAssetAmount(asset, strategies[asset].sharesToUnderlyingView(queuedShares));
```

Comment:
Change this loop to call toUserAssetAmount only once by summing

the results of queuedShares = node.queuedShares(strategies[asset]); and 


```
                uint256 strategyBalance = toUserAssetAmount(
                    asset,
                    strategies[asset].userUnderlyingView((address(node)))
                );
                ```

Reply:
https://github.com/yieldnest/yieldnest-protocol/pull/164/commits/8ba709a10803c2e4c17771a41bae84f4ad2d4639